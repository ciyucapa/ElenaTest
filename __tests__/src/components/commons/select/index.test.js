import React from 'react';
import {create} from 'react-test-renderer';

import Select from '../../../../../src/components/commons/select';

describe('Select component', () => {
    test('render select component', () => {
        const tree = create(
            <Select
                value={'A'} 
                onChange={() => {}}  
                options={[
                    {
                        value: 'A',
                        label: 'CASE A',
                    },
                    {
                        value: 'B',
                        label: 'CASE B',
                    },
                ]}
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});