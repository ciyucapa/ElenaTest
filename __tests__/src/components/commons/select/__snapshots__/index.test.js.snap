// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select component render select component 1`] = `
<select
  onChange={[Function]}
  value="A"
>
  <option
    value="A"
  >
    CASE A
  </option>
  <option
    value="B"
  >
    CASE B
  </option>
</select>
`;
